### PR TITLE
Grammar fixes in README.md

### PR DESCRIPTION
**Pull request by Fawxy**
Small changes to grammar.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 > Luft, the Swedish(and German and Danish) word for air.
 
-Luft aims helps your write shorter, lighter and more readable view controllers. Luft colors your Xcode gutter in a nice green when you're view controller is light. When it start to get a bit heavier the gutter turns yellow to warn you that it's time to start moving code to other places. If you ignore the warning the gutter eventually turns red as you add more code.
+Luft aims helps your write shorter, lighter and more readable view controllers. Luft colors your Xcode gutter in a nice green when your view controller is light. When it starts to get a bit heavier, the gutter turns yellow to warn you that it's time to start moving code to other places. If you ignore the warning the gutter eventually turns red as you add more code.
```
Comment (bogren) [inline README.md]: Can you fix the first line too? :) `Luft aims to help you`

**Comment by k0nserv** [comment]
Thanks a bunch. English grammar is the bane of my existence 😨

**Comment by Fawxy** [comment]
No problem - nice plugin!

**Comment by k0nserv** [comment]
Thanks

**Pull request merged**